tests/providers/test_date_time.py

```diff
@@ -716,3 +718,29 @@ def test_month(self):
         month = self.fake.month_name()
         assert isinstance(month, str)
         assert month in TaInProvider.MONTH_NAMES.values()
+
+
+class TestRuRu(unittest.TestCase):
+    """ Tests date_time in the ru_RU locale """
+
+    def setUp(self):
+        self.fake = Faker('ru_RU')
+        Faker.seed(0)
+
+    def test_day(self):
+        for _ in range(100):
```

Comment:
I think looping 100 times might be excessive, as there are only 7 possible values. Let's do `range(50)`

Reply:
Okay, I reduced it to 50